tests/functional/helpers/local-monorepo.ts

```diff
@@ -159,7 +159,27 @@ export default class LocalMonorepo<
   protected async afterCreate() {
     await super.afterCreate();
 
-    await this.writeJsonFile('package.json', { private: true });
+    await this.writeJsonFile('package.json', {
+      private: true,
+      packageManager: 'yarn@3.2.1',
+    });
+
+    const constraintsProContent = `
```

Comment:
Hmm... this constraints file doesn't really check anything I don't think? I believe that Yarn looks for certain predicates to exist. Perhaps this just file just needs to be present?

Reply:
It checks that each package has a name and a version field
it's required for the constraints fix otherwise it throws `This package doesn't seem to be present in your lockfile; run "yarn install" to update the lockfile`

Reply:
That's a strange error when the constraints file doesn't exist. I would have expected something like "constraints.pro not found".

I just tried removing these lines and running the tests and they don't seem to fail with the error you describe 🤔

Reply:
we can remove the constraints file only the plugin installation is required, but don't we want to test that constraints check really works as this constraints at least checks that each package has a version and name, we can add more check ?